installer/roles/local_docker/tasks/compose.yml

```diff
@@ -10,6 +10,21 @@
     dest: "{{ docker_compose_dir }}/docker-compose.yml"
   register: awx_compose_config
 
+- name: Render secrets file
+  template:
+    src: environment.sh.j2
+    dest: "{{ docker_compose_dir }}/environment.sh"
```

Comment:
Maybe this files better to be `mode: '400'` or `mode: '600'` ?

Reply:
@sumkincpp thanks, addressed here https://github.com/ansible/awx/pull/3515